torch/csrc/api/include/torch/jit.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <string>
```

Comment:
The  header include order should ideally be from most specific to least specific, i.e. your library -> third party libraries -> standard headers, so this would best be

```
#include <torch/csrc/jit/script/module.h>
#include <torch/csrc/jit/script/compiler.h>
#include <torch/csrc/jit/stack.h>

#include <memory>
#include <string>
#include <vector>
```